Resolve #7

afmagee42 opened the pull request:
This PR
1. Amends the likelihood surface explainer to be more explicit about what the shape of the likelihood is and how that drives the convergence issues.
2. Fills in several leaps in logic in the explanation of "how much do these $R$ distributions overlap?"
3. Removes out-of-date text about divergent transitions.

This should resolve the issues with clarity that @cherz4 raised in #7.

cherz4 commented on vignettes/nbbp.Rmd:
```diff
@@ -243,10 +254,17 @@ r_df |>
-(This is equivalent to finding $q$ such that $\text{Pr}(R_{\text{US}} \leq q \mid \text{data}) + \text{Pr}(R_{\text{Canada}} \leq q \mid \text{data}) = 1$, and we can use either posterior to then compute $\alpha$.)
+We might, for example, contemplate what percent  of the posterior on $R_{\text{US}}$ is above the 2.5th percentile of the posterior on $R_{\text{Canada}}$.
+We would then compute the 2.5th percentile of $R_{\text{Canada}}$, find that it is `r {unname(round(r_df |> dplyr::filter(location == "Canada") |> dplyr::pull(r) |> quantile(probs = 0.025), 2))}`, and count the percent of samples of $R_{\text{US}}$ which are larger than this and find that it is `r {round(mean(r_df |> dplyr::filter(location == "US") |> dplyr::pull(r) >= r_df |> dplyr::filter(locat ...
+Or we could do this the other way around, and ask what percent  of the posterior on $R_{\text{Canada}}$ is below the which is below the 97.5th percentile of the posterior on $R_{\text{US}}$.
```
Change to 'is below the 97th percentile. There are some extra words in the sentence

cherz4 reviewed (approved):
Thanks for addressing these.

Questions/Potential to be resolveds:
1) Is the second likelihood surface plot supposed have different values on the x and y axes? The heatmap has changed (and the legend) but not the axes and that is throwing off my understanding of your point.
2) Have placed a comments for one easy to fix typo.

Nits that doesn't impact merging this in:
1) "the region where the log-likelihood is within 12 units of the maximum value is within a bivariate confidence region" - would you be able to link to a reference for where this limit comes from?

afmagee42 commented:
> Questions/Potential to be resolveds:
> 
> 1. Is the second likelihood surface plot supposed have different values on the x and y axes? The heatmap has changed (and the legend) but not the axes and that is throwing off my understanding of your point.

The second plot is for $0 \leq R \leq 0.02$ where the first was $0 \leq R \leq 2$. I have added text to explain that the second plot focuses on small $R$.

> Nits that doesn't impact merging this in:
> 
> 1. "the region where the log-likelihood is within 12 units of the maximum value is within a bivariate confidence region" - would you be able to link to a reference for where this limit comes from?

I've clarified that this comes as a result of Wilks' theorem. And I realized there was an actual issue here that I had missed, which is that 12 is specifically for a 95% CI, so I added that.
